tests/unit/module_utils/elbv2/test_elbv2.py

```diff
@@ -5,161 +5,171 @@
-        assert self.elbv2obj.changed is False
-
-    # Test modify_ip_address_type
-    def test_modify_ip_address_type_update(self):
-        # Run module
-        self.elbv2obj.modify_ip_address_type("dualstack")
-        # check that no method was called and this has been retrieved from elb attributes
-        self.connection.set_ip_address_type.assert_called_once()
-        # assert we got the expected value
-        assert self.elbv2obj.changed is True
-
-    # Test get_elb_attributes
-    def test_get_elb_attributes(self):
-        # Build expected result
-        expected_elb_attributes = {
-            "access_logs_s3_bucket": "",
-            "access_logs_s3_enabled": "false",
-            "access_logs_s3_prefix": "",
-            "deletion_protection_enabled": "false",
-            "idle_timeout_timeout_seconds": "60",
-            "routing_http2_enabled": "true",
-            "routing_http_desync_mitigation_mode": "defensive",
-            "routing_http_drop_invalid_header_fields_enabled": "true",
-            "routing_http_x_amzn_tls_version_and_cipher_suite_enabled": "true",
-            "routing_http_xff_client_port_enabled": "true",
-            "waf_fail_open_enabled": "true",
-        }
-        # Run module
-        actual_elb_attributes = self.elbv2obj.get_elb_attributes()
-        # Assert we got the expected result
-        assert actual_elb_attributes == expected_elb_attributes
-
-    @pytest.mark.parametrize(
-        "subnets,subnet_mappings,elb_subnets,expected",
-        [
-            (
-                None,
-                [{"SubnetId": "subnet-abc", "AllocationId": "eipalloc-abc"}],
-                [{"SubnetId": "subnet-abc", "LoadBalancerAddresses": [{"AllocationId": "eipalloc-abc"}]}],
-                True,
-            ),
-            (
-                ["subnet-abc"],
-                None,
-                [{"SubnetId": "subnet-abc", "LoadBalancerAddresses": [{"AllocationId": "eipalloc-abc"}]}],
-                False,
-            ),
-            (["subnet-abc"], None, [{"SubnetId": "subnet-abc"}], True),
-            (
-                None,
-                [
-                    {"SubnetId": "subnet-123", "AllocationId": "eipalloc-123"},
-                    {"SubnetId": "subnet-abc", "AllocationId": "eipalloc-abc"},
-                ],
-                [
-                    {"SubnetId": "subnet-abc", "LoadBalancerAddresses": [{"AllocationId": "eipalloc-abc"}]},
-                    {"SubnetId": "subnet-123", "LoadBalancerAddresses": [{"AllocationId": "eipalloc-123"}]},
-                ],
-                True,
-            ),
-        ],
-    )
-    def test_compare_subnets(self, subnets, subnet_mappings, elb_subnets, expected):
-        self.elbv2obj.subnets = subnets
-        self.elbv2obj.subnet_mappings = subnet_mappings
-        self.elbv2obj.elb = {"AvailabilityZones": elb_subnets}
-        assert expected == self.elbv2obj.compare_subnets()
+# class TestElBV2Utils:
```

Comment:
Do we need to keep this commented out section?

Reply:
No, I will remove it.